MonoGame.Framework.Content.Pipeline/Audio/AudioContent.cs

```diff
@@ -134,229 +114,188 @@ int QualityToBitRate(ConversionQuality quality)
+                    throw new InvalidOperationException("ffmpeg exited with non-zero exit code: \n" + ffmpegStdout + "\n" + ffmpegStderr);
+                }
 
-                case ConversionFormat.ImaAdpcm:
-#if WINDOWS
-                    ConvertWav(new ImaAdpcmWaveFormat(QualityToSampleRate(quality), format.ChannelCount, 4));
-                    break;
-#else
-                    throw new NotImplementedException("ImaAdpcm has not been implemented on this platform");
-#endif
+                byte[] rawData;
+                using (var fs = new FileStream(temporaryOutput, FileMode.Open, FileAccess.Read))
+                {
+                    rawData = new byte[fs.Length];
+                    fs.Read(rawData, 0, rawData.Length);
+                }
 
-                case ConversionFormat.Aac:
-#if WINDOWS
-                    reader.Position = 0;
-                    var mediaType = SelectMediaType (AudioSubtypes.MFAudioFormat_AAC, reader.WaveFormat, QualityToBitRate (quality));
-                    if (mediaType == null) {
-                        throw new InvalidDataException ("Cound not find a suitable mediaType to convert to.");
-                    }
-                    using (var encoder = new MediaFoundationEncoder (mediaType)) {
-                        encoder.Encode (targetFileName, reader);
-                    }
-                    break;
-#elif LINUX
-                    // TODO: Code for Linux convertion
-                    throw new NotImplementedException("Aac has not been implemented on this platform");
-#else //MONOMAC
-                    if (!ConvertAudio.Convert(fileName, targetFileName, AudioFormatType.MPEG4AAC, MonoMac.AudioToolbox.AudioFileType.MPEG4, quality)) {
-                        throw new InvalidDataException("Failed to convert to AAC");
+                if (saveToFile != null)
+                {
+                    using (var fs = new FileStream(saveToFile, FileMode.Create, FileAccess.Write))
+                    {
+                        fs.Write(rawData, 0, rawData.Length);
                     }
-                    break;
-#endif
 
-                case ConversionFormat.Vorbis:
-                    throw new NotImplementedException("Vorbis is not yet implemented as an encoding format.");
-            }
-        }
+                    this.data = null;
+                }
+                else
+                {
+                    this.data = rawData.ToList();
+                }
 
-        /// <summary>
-        /// Immediately releases the managed and unmanaged resources used by this object.
-        /// </summary>
-        public void Dispose()
-        {
-            Dispose(true);
-            GC.SuppressFinalize(this);
-        }
+                string ffprobeStdout, ffprobeStderr;
+                var ffprobeExitCode = ExternalTool.Run(
+                    "ffprobe",
+                    string.Format("-i \"{0}\" -show_entries streams -v quiet -of flat", temporarySource),
```

Comment:
Note we're probing the original source file and not the converted output file here.  This means the format is likely incorrect.